Buildings/Fluid/Air/AirHandlingUnit.mo

```diff
@@ -0,0 +1,258 @@
+    "Range of significant deviation from equal percentage law"
+    annotation(Dialog(group="Valve"));
+
+  // Parameters for electric heater
+  parameter Modelica.SIunits.Time tauEleHea = 30
+    "Time constant at nominal flow for electric heater (if energyDynamics <> SteadyState)"
+     annotation (Dialog(tab = "Dynamics", group="Electric heater",
+       enable=not (energyDynamics==Modelica.Fluid.Types.Dynamics.SteadyState)));
+  parameter Modelica.SIunits.Efficiency etaHea = 1.0 "Efficiency of electrical heater"
+    annotation (Dialog(group="Electric heater"));
+  parameter Modelica.SIunits.HeatFlowRate QHeaMax_flow(min=0)
+    "Nominal heating capacity of eletric heater,positive"
+    annotation (Dialog(group="Electric heater"));
+  // Parameters for humidifier
+  parameter Modelica.SIunits.MassFlowRate mWatMax_flow(min=0)
+    "Nominal humidification capacity for humidifier, positive for humidification"
+    annotation (Dialog(tab="General",group="Humidifier"));
+  parameter Modelica.SIunits.Temperature THum = 293.15
+    "Temperature of water that is added to the fluid stream by the humidifier"
+    annotation (Dialog(group="Humidifier"));
+  parameter Modelica.SIunits.Time tauHum = 30
+    "Time constant at nominal flow for humidifier(if energyDynamics <> SteadyState)"
+     annotation (Dialog(tab = "Dynamics", group="Humidifier",
+       enable=not (energyDynamics==Modelica.Fluid.Types.Dynamics.SteadyState)));
+
+  // parameters for heater controller
+  parameter Real yValLow(min=0, max=1, unit="1")=0.1
+  "if yVal<=yValLow, hysVal.y switches to false"
+  annotation(Dialog(group="Reheat controller"));
+  parameter Real yValHig(min=0, max=1, unit="1")=0.15
+  "if yVal>=yValHig, hysVal.y switches to true"
+  annotation(Dialog(group="Reheat controller"));
+  parameter Modelica.SIunits.TemperatureDifference dTLow=-0.5
+  "if dT<=dTLow, hysTemDif.y switches to false"
+  annotation(Dialog(group="Reheat controller"));
+  parameter Modelica.SIunits.TemperatureDifference dTHig=0.5
+  "if dT>=dTHig, hysTemDif.y switches to true"
+  annotation(Dialog(group="Reheat controller"));
+  parameter Boolean pre_yVal_start=true "Previous value of hysVal.y used at initialization"
+    annotation (Dialog(group="Reheat controller", tab="Initialization"));
+  parameter Boolean pre_dT_start=true "Previous value of hysTemDif.y used at initialization"
+    annotation (Dialog(group="Reheat controller", tab="Initialization"));
+
+  Modelica.Blocks.Interfaces.RealOutput PHea(unit="W")
+    "Power consumed by electric heater" annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=-90,
+        origin={18,-110})));
+  Modelica.Blocks.Interfaces.RealInput TSet(unit="K")
+    "Set point temperature of the fluid that leaves port_b" annotation (
+      Placement(transformation(extent={{-140,-30},{-100,10}}),
+        iconTransformation(extent={{-120,-10},{-100,10}})));
+  Modelica.Blocks.Interfaces.RealInput XSet_w(unit="kg/kg")
```

Comment:
`unit=1`, the same unit setting is used in model `Buildings.Fluid.Humidifiers.SteamHumidifier_X`.